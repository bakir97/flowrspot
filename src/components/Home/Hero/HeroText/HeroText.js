import React from "react";
import styles from "./HeroText.module.scss";
const HeroText = () => {
  return (
    <div className={styles.container}>
      <h1 className={styles.main_text}>Discover flowers around you</h1>
      <p className={styles.sub_text}>
        Explore between more than 8.427 sightings
      </p>
    </div>
  );
};

export default HeroText;
