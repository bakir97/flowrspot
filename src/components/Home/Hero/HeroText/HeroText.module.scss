@import "../../../../assets/global/styles/colors.scss";
@import "../../../../assets/global/styles/mixins.scss";
.container {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  margin-bottom: 45px;
}
.main_text {
  @include font-light;
  margin-bottom: 15px;
  font-size: 40px;
}
.sub_text {
  @include font-light;
  opacity: 0.7;
  font-size: 17px;
}
