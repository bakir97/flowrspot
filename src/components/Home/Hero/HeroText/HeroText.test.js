import React from "react";
import { shallow } from "enzyme";
import HeroText from "./HeroText";
describe("HeroText", () => {
  it("should render correctly", () => {
    const component = shallow(<HeroText />);
    expect(component).toMatchSnapshot();
  });
});
