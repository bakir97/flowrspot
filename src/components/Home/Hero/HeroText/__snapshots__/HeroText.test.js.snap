// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroText should render correctly 1`] = `
<div
  className="container"
>
  <h1
    className="main_text"
  >
    Discover flowers around you
  </h1>
  <p
    className="sub_text"
  >
    Explore between more than 8.427 sightings
  </p>
</div>
`;
